### ISSUE TITLE
get a players gamemode

### ISSUE THREAD

**Issue by notanoperator** [TotalFreedom/TotalFreedomMod#1839]
Simple. /getgamemode. Get's the player's gamemode so that administrative action may be used :D 
I already made the command and tested it, just wanted thoughts (and you have to post an issue before making a pr.)

**Comment by notanoperator** [comment]
@creeperseth, but some admins don't want to see all of the other spammy stuff, and plus, while looking through all of the whois bs, the player could have already changed there gamemode.

**Comment by Wild1145** [comment]
I agree with @CreeperSeth , I feel this would be redundant information and as I said in the other issue that you mentioned this, it takes a lot longer to type out your command than someone to type /survival or /creative to fool you.

**Comment by notanoperator** [comment]
Aliases "isgmc,getgm,gym"

**Comment by jjkatz** [comment]
Well based on his, maybe we should remove some of the unneeded info from /whois

**Comment by aggelosQQ** [comment]
@notanoperator it doesn't matter if they change their gamemode as long as you have done /whois to the subject user, you'll be able to see their gamemode at the time you executed the command.

**Comment by foxcqrn** [comment]
This is useless and very easy to create. We already have /whois

**Comment by Wild1145** [comment]
Any other comments to add?
